### ISSUE TITLE
Vagrant up in Konsole eats my input (example gif included)

### ISSUE THREAD

**Issue by xenithorb** [mitchellh/vagrant#8144]
### Vagrant version

1.9.1

### Host operating system

Fedora 25 KDE

### Expected behavior

Vagrant should not affect the well being of the current terminal as far as consuming input characters. 

### Actual behavior

`vagrant up` (only when a vm actually begins running, to include remotely with libvirt) consumes every other keystroke from the `konsole` commandline. In layman's terms every time I vagrant up a box to check something out, I have a to switch to a new terminal because it's impossible to work on that terminal now. 

### Steps to reproduce

1. Get Fedora KDE and install Vagrant as normal
2. Open Konsole
3. Vagrant up somewhere and let it terminate itself (whether that's an error from the provisioner or a successful ending, as long as you haven't yet done `vagrant destroy` it will appear)
4. Observe that some of your input is now missing, giving the effect that your keyboard is broken.

### Other

 - My apologies that it only seems to happen in konsole, but a bug's a bug right? I really don't know where else to put this and I believe it has something to do with some errant pipe redirecting. 

### Example gif:

![outfile](https://cloud.githubusercontent.com/assets/10273878/21370522/0c6d6f24-c6da-11e6-93fc-5419555ca28c.gif)

You might be inclined to think, "oh, but it's just CPU usage or something," but the thing is the VM I launced is remote and not on the vagrant control system. In otherwords, this happens regardless of the proximity of the VM.

**Comment by xenithorb** [comment]
I also forgot to mention the most important and empirical part!

If you run vagrant with `vagrant up <box> </dev/null` it also doesn't seem to occur.

**Comment by richard-scott** [comment]
Have you tried typing 'reset' after your 'vagrant up' command.
That will reset your console.

**Comment by chrisroberts** [comment]
Have you attempted https://github.com/mitchellh/vagrant/issues/8144#issuecomment-271257518 and does it make any difference? Also, can you provide the Vagrantfile that you are using to cause this issue? Thanks!

**Comment by chrisroberts** [comment]
Hey there,

I am going to close this due to lack of response. If this is still occurring, please open a [_new issue_](https://github.com/mitchellh/vagrant/issues/new) and follow the provided issue template that appears when you click the "New Issue" button. This will help us in getting a reproduction and fix. Thanks! 😄

**Comment by xenithorb** [comment]
I stopped using vagrant for some time because of this but went back to development on something that I needed to use it for again and I can at least report that it no longer does this on Fedora 27 with vagrant version 1.9.8.
